packages/app/src/systems/Asset/hooks/useAsset.tsx

```diff
@@ -0,0 +1,14 @@
+import { ASSET_LIST } from "../utils";
+
+export function useAsset(assetId: string) {
+  const asset = ASSET_LIST.find((asset) => asset.assetId === assetId);
```

Comment:
You can use an `useMemo` here with the `assetId` to avoid unnescessary recomputations